### DISCUSSION TITLE
How to receive LVARS outside of the simulator?

### DISCUSSION THREAD

**Discussion by speedlao** [flybywiresim/aircraft#5970] [Q&A] [answered]
Dear FBW dev team & communities,

I'm creating my arduino based cockpit gauges, since I studied from this thread (https://github.com/flybywiresim/a32nx/discussions/4518) that Local-Vars are unabled to be accessed via SimConnect.
Is there any official method for me to those variables?
  - GET and SET
  - PIPE or TCP/UDP server?
  - Public interface from FBW for 3rd-party develop needs?

Cheers and have a nice flight!

**Comment by aguther** [comment, marked as answer]
No, LVARs are not accessible via SimConnect. That is the reason why FSUIPC and SPAD.next for example have LVAR bridges that need to be installed into the sim. You could consider using the FSUIPC API to achieve what want.

**Reply by speedlao** [reply]
Got it! and thanks for the reply. Going to study WASM mechanism.

Cheers

**Comment by igor8518** [comment]
I'm working on it right now. I'm implementing some API to A32NX.

**Reply by Peter-Mary-Jo** [reply]
Don't you think  that LVARS can be accessed via AAO?

**Reply by igor8518** [reply]
I don't know what AAO is.) I'm creating an interface like the PMDG SDK.

**Reply by Peter-Mary-Jo** [reply]
AAO is an application that is a bridge between devices and the simulator with which you can write complex script and also use windows voice recognition. 
https://axisandohs.weebly.com/
you can find template for A32nx on github

**Reply by igor8518** [reply]
Thank you, I understand. But I write my code a bit for other tasks.
